### ISSUE TITLE
Refused to apply inline style because it violates the following Content Security Policy

### ISSUE THREAD

**Issue by JandosHk** [HangfireIO/Hangfire#1897]
I started using AspNet.Core 3.0 endpoints, and got CSP error. Apparently, the dashboard contains some inline styles such as

    <div class="container" style="margin-bottom: 20px;">

What should I do?

**Comment by JandosHk** [comment]
Thank you @odinserj. Not to forget, I'm closing the issue. BTW, I added my star to Hangfire.

**Comment by odinserj** [comment]
Please leave it open to help me track it. I will make this and other changes to move everything to CSS and release in 1.7.25, but next month only.

**Comment by JandosHk** [comment]
Okay.

**Comment by odinserj** [comment]
Thanks for your patience. `style` attribute usage was removed in commit https://github.com/HangfireIO/Hangfire/commit/0c816995ab480735a7d560abd131b6875f6c312b, so they will not be used for the primary pages in the upcoming 1.7.25. I've scheduled feature to include third-party CSS files to make this also work for extensions, but it will only be possible once 1.8.0 is released, and extension packages (like batches or throttlers) will be republished, so it will take much more time, and unfortunately meanwhile the only possible solution will be to enable inline styles in CSP.
